### ISSUE TITLE
AISHU RANU

### ISSUE THREAD

**Issue by ranveersingh7** [ranveersingh7/practice-git-1#2]
# ~~**FIRST KISS**~~  **FIRST GOOD KISS**
_Even before I realised you loved me too and_

**Comment by ranveersingh7** [comment]
vbh

**Comment by ranveersingh7** [comment]
> # ~~**FIRST KISS**~~  **FIRST GOOD KISS**

HEY

**Comment by ranveersingh7** [comment]
> # ~~**FIRST KISS**~~  **FIRST GOOD KISS**
> 
_Even before I realised you loved me too and you wanted this relationship to happen I knew due to the connection we shared that someday I am going to have a most beautiful hug and most beautiful kiss with you.. ❤❤❤

I was sure that kiss will just be natural and loving...

- [ ] FIRST ATTEMPT
And finally that day comes..
The day I was sure kiss would happen and I was sonready for it "OUR MOVIE DATE" 

I moved ahead to kiss my girl and REJECTED. 🤣

- [ ] _SECOND ATTEMPT
REJECTED 
 Next movie date sooo beautiful.
We having conversations with nose touched and completely into each other and you tell me we should not kiss untill the moment is right. I was pleased after controlling myself soo much it was possible to not kiss.

That day I was sure about how thoughtful and sensitive of a person you were as the moment and 

- [ ] ~~KISS~~ PECK
